cmd/process-agent/runtimecfg/runtime_config_test.go

```diff
@@ -0,0 +1,98 @@
+package runtimecfg
+
+import (
+	ddconfig "github.com/DataDog/datadog-agent/pkg/config"
+	"github.com/DataDog/datadog-agent/pkg/process/config"
+	"gopkg.in/yaml.v3"
+	"testing"
+	"time"
+)
+
+var cfgClient *ProcessAgentRuntimeConfigClient
+
+func init() {
+	// Set up the logger, so we can test using log_level
+	cfg, err := config.NewAgentConfig("", "", "")
+	if err != nil {
+		panic(err)
+	}
+	if err = cfg.LoadProcessYamlConfig(""); err != nil {
+		panic(err)
+	}
+
+	const rpcPort = "1234"
+
+	// Start the RPC Server
+	if err := StartRuntimeSettingRPCService(rpcPort); err != nil {
+		panic(err)
+	}
+
+	// Give the RPC Server a second to start up
+	time.Sleep(time.Second)
```

Comment:
Can this be done differently?

Reply:
I actually spent a good bit of time trying to see if there was a different way to do this, but I couldn't find one. The problem here is that http.Serve is blocking and never exits unless there is an error, so it's not really possible to signal to the main thread when it has begun.

I will investigate this further